packages/forms/src/reducer.ts

```diff
@@ -92,7 +92,7 @@ const behaviors: Record<FormActionType, Function> = {
     source: state.model,
     showAllErrors: true,
   }),
-  [XCRITICAL_FORM_SHOW_ERRORS]: (state: IFormState, { payload }) => ({
+  [XCRITICAL_FORM_SHOW_ERRORS]: (state: IFormState, { payload }: any) => ({
```

Comment:
why do you use "any" here? i think, it must be boolean (because we either show all errors or don't)